COMDOX-244: Migrate Frontend Developer Guide from magento/devdocs (#7)
AdobeDocs/commerce-frontend-core: src/pages/guide/themes/apply-admin.md

---
title: Apply an Admin Theme | Commerce Frontend Development

> description: View code samples for applying a theme for the Adobe Commerce and Magento Open Source Admin application.

Reviewer: ```suggestion
description: Learn how to apply an Admin theme to the Adobe Commerce and Magento Open Source Admin user interfaces.
```
